src/components/profile/__tests__/__snapshots__/ProfilePrivacy.js.snap

```diff
@@ -945,49 +945,19 @@ exports[`ProfilePrivacy matches snapshot 1`] = `
               className="css-view-1dbjc4n"
               style={
                 Object {
-                  "WebkitFlexBasis": "auto",
                   "borderBottomLeftRadius": "42.38961038961039px",
                   "borderBottomRightRadius": "42.38961038961039px",
                   "borderTopLeftRadius": "42.38961038961039px",
                   "borderTopRightRadius": "42.38961038961039px",
-                  "flexBasis": "auto",
                   "height": "84.77922077922078px",
-                  "msFlexPreferredSize": "auto",
-                  "overflowX": "hidden",
-                  "overflowY": "hidden",
                   "position": "absolute",
                   "top": "-42.38961038961039px",
                   "width": "84.77922077922078px",
                   "zIndex": 1,
                 }
               }
             >
-              <div
-                className="css-view-1dbjc4n"
-                style={
-                  Object {
-                    "backgroundColor": "rgba(0,0,0,0.00)",
-                    "backgroundImage": "url(\\"unknownProfile.svg\\")",
-                    "backgroundPosition": "center",
-                    "backgroundRepeat": "no-repeat",
-                    "backgroundSize": "cover",
-                    "bottom": "0px",
-                    "height": "100%",
-                    "left": "0px",
-                    "position": "absolute",
-                    "right": "0px",
-                    "top": "0px",
-                    "width": "100%",
-                    "zIndex": -1,
-                  }
-                }
-              />
-              <img
-                alt=""
-                className="css-accessibilityImage-9pa8cd"
-                draggable={false}
-                src="unknownProfile.svg"
-              />
+              <unknownProfile.svg />
```

Comment:
looks like filename substituted as generated component's name
we have to adjust it to the React convention - remove .svg extension & uppercase first character

Reply:
(do not fix it!)